hp/elitebook/845/g7/default.nix

```diff
@@ -0,0 +1,25 @@
+{ config, pkgs, lib, ... }:
```

Comment:
Please also add this to the flake.nix and the README.md table.
Otherwise CI won't pick up your model into our testing configuration.

Reply:
Thanks @Mic92, I've updated the branch with those adjustments.